src/Generics/Utils/Ext.agda

```diff
@@ -0,0 +1,104 @@
+{-# OPTIONS --safe #-}
+module Generics.Utils.Ext where
+
+open import Prelude
+open import Data.Nat using (_≤ᵇ_)
+
+open import Class.Show.Core; open import Class.Show.Instances
+open import Class.Semigroup.Core; open import Class.Semigroup.Instances
+open import Class.Monad.Core; open import Class.Monad.Instances
+open import Class.Applicative.Core; open import Class.Applicative.Instances
+
+open import Interface.Functor using (map; _<$>_; Type↑; Functor)
+
+open import Generics
+open import Agda.Builtin.Reflection using (getInstances)
+open import Reflection hiding (_>>=_; _>>_)
+open import Reflection.AST.Term
+
+-- N-ary extension of the TC context.
+extendContextTel : ∀ {A : Set} → Telescope → TC A → TC A
```

Comment:
This is in stdlib-meta too

Reply:
It turns out that `MonadTC.extendTel'` does not include the names which is unfortunate; I propose we fix it upstream and then remove this when we bump stdlib-meta version.